build.go

```diff
@@ -625,18 +639,39 @@ func randString(n int) string {
 	return string(b)
 }
 
-func buildPlatformVolumes(volumes []string) ([]string, error) {
+func buildPlatformVolumes(imgOS string, volumes []string) ([]string, error) {
```

Comment:
Could we add in a unit tests verifying that the volume is constructed appropriately?